packages/clerk-js/src/ui/components/OrganizationProfile/DomainList.tsx

```diff
@@ -82,23 +132,12 @@ export const DomainList = withGate(
                 })}
                 badge={<EnrollmentBadge organizationDomain={d} />}
                 trailingComponent={
-                  <ThreeDotsMenu
-                    actions={[
-                      {
-                        label: localizationKeys(
-                          'organizationProfile.profilePage.domainSection.unverifiedDomain_menuAction__verify',
-                        ),
-                        onClick: () => navigate(`${redirectSubPath}${d.id}/verify`),
-                      },
-                      {
-                        label: localizationKeys(
-                          'organizationProfile.profilePage.domainSection.unverifiedDomain_menuAction__remove',
-                        ),
-                        isDestructive: true,
-                        onClick: () => navigate(`${redirectSubPath}${d.id}/remove`),
-                      },
-                    ]}
-                  />
+                  showDotMenu ? (
```

Comment:
🙃 `showDotMeny && <JSX/>`

Reply:
Already tried that, seems like `trailingComponent` does not accept null, I didn't want to mess with the signature of the component.

Reply:
The following has type of ` false | EmotionJSX.Element | null`

```tsx
showDotMenu && (
                    <DomainListDotMenu
                      redirectSubPath={redirectSubPath}
                      domainId={d.id}
                    />
                  )


```

So i would have to do 
```tsx

(showDotMenu  || null ) && (
                    <DomainListDotMenu
                      redirectSubPath={redirectSubPath}
                      domainId={d.id}
                    />
                  )
```

Which i'm not convinced it is better

Reply:
I am not sure I understand what you are talking about. Is this [the](https://github.com/clerkinc/javascript/blob/b2296d6304e1ca31a35450e0c67a12555c0142f9/packages/clerk-js/src/ui/elements/BlockWithTrailingComponent.tsx#L10) type?

Reply:
This is the TS error 
<img width="1078" alt="image" src="https://github.com/clerkinc/javascript/assets/19269911/d051f64e-1c3d-45f0-bcbe-94f186f7eb8e">